rollingpin/main.py

```diff
@@ -137,8 +137,8 @@ def _select_hosts(config, args):
     hosts = interleaved(all_hosts_unsorted, key=lambda h: h.pool)
 
     try:
-        aliases = resolve_aliases(config["aliases"], hosts)
-        full_hostlist = resolve_hostlist(args.host_refs, hosts, aliases)
+        full_hostlist = resolve_hostlist(
+            args.host_refs, hosts, config["aliases"])
```

Comment:
No, this is just getting data out of the config dict built [up here](https://github.com/reddit/rollingpin/blob/master/rollingpin/main.py#L114) and is unrelated to the name of the section in the config file.